pipeline/mutate/mutator_id_token.go

```diff
@@ -70,10 +70,6 @@ func (a *MutatorIDToken) WithCache(t *template.Template) {
 	a.templates = t
 }
 
-func (a *MutatorIDToken) SetCaching(token bool) {
-	a.tokenCacheEnabled = token
-}
-
 type idTokenCacheContainer struct {
 	ExpiresAt time.Time
```

Comment:
I haven't looked at this code in detail, but `ExpiresAt`+`TTL` seem to be semantically the same thing? Do we even need to store these in the cache?

Reply:
Good catch, we don't!

I think that's what was buggy in the first place: we don't set an actual TTL in the cache, but instead cache the TTL along with the token.

I pushed edit 👍